sources/addr.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "addr",
 	"name": "addr",
 	"prefix": "addr",
-    	"key":"id",
+	"key": "id",
 	"url": "",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "N/A",
+	"license_type": "CC0",
```

Comment:
Is there a `src:addr` we sometimes use? In any event, the values in `addr` may or may-not be CC0, so let's leave this either blank or `N/A`.